### DISCUSSION TITLE
Propose: localized group categories

### DISCUSSION THREAD

**Discussion by Dum4G** [iptv-org/community#2] [Ideas]
A new folder "strings" will be created in root of the repo and translation files for categories will be placed in there

For country playlists each category will be named in both English and the most common language of a country like so "Entertainment / Развлекательные" or in any other viable form for both foreigners and native viewers

For language playlists only localized variant will be used since viewers already target some language they know.

Translations will be gathered and automated for free from Crowdin as they're offering free plan for open-source projects with comfortable tools or directly commited by contributors.

Inspired by my father who actually asked me to translate all categories

**Comment by chamberitiger** [comment]
Algún canal español? Para iptv

**Comment by BellezaEmporium** [comment]
It sounds interesting in my opinion, and it's pretty easy to put in place.

-> On Crowdin, get the ISO 3166 country names.
-> Make the script pull the translations from Crowdin via a fetch and simply translate the groups via a "string match".

**Comment by Dum4G** [comment]
They're already offering Github integration and probably support ISO 3166 country names.

**Comment by BellezaEmporium** [comment]
Even better then :)

**Comment by Dum4G** [comment]
I hate when ideas get stale. Are you ready to try to participate in and then enhance your building scripts? @freearhey 
https://crowdin.com/page/open-source-project-setup-request

**Reply by freearhey** [reply]
No, I don't have time for that right now.

**Reply by Dum4G** [reply]
> No, I don't have time for that right now.

Any progress?

**Reply by freearhey** [reply]
I personally haven't worked on it yet.
